content/en/_index.html

```diff
@@ -296,9 +296,11 @@ <h5>November 21, 2022</h5>
       <div class="tag">Blog</div>
     </div>
     <p class="font-size-sm">
-      Notary and Notation Notary project is a set of tools that helps you sign,
+      Notation is a CLI and set of libraries that helps you sign,
       store, and verify OCI artifacts using OCI-conformant registries. Notation
-      is an implementation of the Notary v2 specifications.
+      is an implementation of the Notation specification. The Notary Project is 
```

Comment:
Notation is an implementation of the Notary project specification

Reply:
I think we should change to: 
> Notation is an implementation of the Notary Project signing specification.

Reply:
fixed